src/components/Dropdown/Dropdown.tsx

```diff
@@ -76,77 +88,81 @@ export interface DropdownProps {
    * Used in addition to labelText for screen readers.
    */
   'aria-labelledby'?: string;
-  /** Visual hint to show if nothing is selected */
-  visualPlaceholder?: ReactNode;
-  /** Change visualPlaceholder by selection
-   * @default true
-   */
-  changeVisualPlaceholderToSelection?: boolean;
   /** Custom classname to extend or customize */
   className?: string;
   /** Properties given to dropdown's Button-component, className etc. */
-  dropdownButtonProps?: OptionalMenuButtonProps;
+  dropdownButtonProps?: OptionalListboxButtonProps;
   /** Properties given to dropdown's popover-component, className etc. */
-  dropdownPopoverProps?: OptionalMenuPopoverProps;
-  menuPopoverComponent?: React.ComponentType<OptionalMenuPopoverProps>;
+  dropdownPopoverProps?: OptionalListboxPopoverProps;
+  listboxPopoverComponent?: React.ComponentType<OptionalListboxPopoverProps>;
   /** Properties given to dropdown's item-component, className etc. */
-  dropdownItemProps?: OptionalMenuItemProps;
+  dropdownItemProps?: OptionalListboxOptionProps;
   /** DropdownItems */
   children?:
     | Array<ReactElement<DropdownPopoverItems>>
     | ReactElement<DropdownPopoverItems>;
+
+  onChange?(newValue: string): void;
 }
 
 export class Dropdown extends Component<DropdownProps> {
-  state: DropdownState = { selectedName: undefined };
+  state: DropdownState = {
+    selectedValue: !!this.props.defaultValue
+      ? this.props.defaultValue
+      : !!this.props.value
+      ? this.props.value
+      : undefined,
```

Comment:
could this just be optionA || optionB || optionC ?

Reply:
Nope, since also non-truthy values need to be supported e.g. empty string.

Reply:
This was changed to honor value over defaultValue even if provided untruthy value e.g. value={undefined}.